packages/core/ios/RNSentryReplay.m

```diff
@@ -56,10 +64,24 @@ + (NSArray *_Nonnull)getReplayRNRedactClasses:(NSDictionary *_Nullable)replayOpt
 
 + (void)postInit
 {
+    // We can't import RNSentryReplayMask.h here because it's Objective-C++
+    // To avoid typos, we test the class existens in the tests
+    [PrivateSentrySDKOnly setRedactContainerClass:[self getMaskClass]];
+    [PrivateSentrySDKOnly setIgnoreContainerClass:[self getUnmaskClass]];
     RNSentryReplayBreadcrumbConverter *breadcrumbConverter =
         [[RNSentryReplayBreadcrumbConverter alloc] init];
     [PrivateSentrySDKOnly configureSessionReplayWith:breadcrumbConverter screenshotProvider:nil];
 }
 
++ (Class) getMaskClass
+{
+    return NSClassFromString(@"RNSentryReplayMask");
```

Comment:
`l`: This is a suggestion since the current approach works as expected and is covered by test cases.
Wdyt of using an objective-c wrapper to get the class names statically? Eg:

```
// RNSentryReplayWrapper.h

@interface RNSentryReplayWrapper : NSObject

+ (Class) getMaskClass;
+ (Class) getUnmaskClass

@end
```

```
// RNSentryReplayWrapper.mm

#import "RNSentryReplayWrapper.h"
#include "RNSentryReplayMask.h"
#include "RNSentryReplayUnmask.h"

@implementation MyObjectiveCWrapper

+ (Class)getMaskClass
{
    return RNSentryReplayMask.class;
}

+ (Class)getUnmaskClass
{
    return RNSentryReplayUnmask.class;
}

@end
```

Reply:
Maybe I've misunderstood but the wrapper it self would not help with the importing Cpp to Obj-C, but I've made different changes that enable us to use the classes directly.

https://github.com/getsentry/sentry-react-native/pull/4224/commits/050acade6e7d3e2d0843dbfbcb1e82e67a713c7b